app.go

```diff
@@ -111,6 +111,21 @@ func (app *App) SetBasicAuth(user, password string) {
 	app.basicAuthPassword = password
 }
 
+// IsBasicAuth indicate authentication is basic or not
+func (app *App) IsBasicAuth() bool {
```

Comment:
@tranvan538 
I think the function name is should be HasBasicAuth

Reply:
Ok, I will change it.